targets/targets.json

```diff
@@ -3682,6 +3682,8 @@
     "NRF52840_DK": {
         "supported_form_factors": ["ARDUINO"],
         "inherits": ["MCU_NRF52840"],
+        "macros_add": ["MBEDTLS_CONFIG_HW_SUPPORT"],
+        "features": ["CRYPTOCELL310"],
```

Comment:
This should go in the `MCU_NRF52840` section since it is MCU specific and not target specific.